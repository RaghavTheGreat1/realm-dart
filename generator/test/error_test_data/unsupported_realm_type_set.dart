import 'package:realm_common/realm_common.dart';

//part 'unsupported_realm_type_set.g.dart';

@RealmModel()
class _Bad {
  @PrimaryKey()
  late int id;

  late Set<_Bad> wrong1;
}
