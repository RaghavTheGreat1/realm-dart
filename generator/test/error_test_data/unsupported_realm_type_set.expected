Set<_Bad> is not supported

in: asset:pkg/test/error_test_data/unsupported_realm_type_set.dart:10:8
    ╷
5   │ @RealmModel()
6   │ class _Bad {
    │       ━━━━ in realm model for 'Bad'
... │
10  │   late Set<_Bad> wrong1;
    │        ^^^^^^^^^ Set element type is not supported
    ╵
Ensure set element type _Bad is a type supported by RealmSet

